timeseries/src/autogluon/timeseries/models/local/abstract_local_model.py

```diff
@@ -82,9 +63,12 @@ def __init__(
             **kwargs,
         )
 
-        self._local_model_args: Dict[str, Any] = None
-        self._seasonal_period: Optional[int] = None
-        self._dummy_forecast: Optional[pd.DataFrame] = None
+        self.n_jobs: int
```

Comment:
if these are indeed "hyperparameters" can they safely be removed as instance variables? I think this is a cause for inconsistency that's currently spread across model implementations so we should get rid of them as much as possible. Ideally, hyperparameters should be fixed during the lifetime of a model and should only be accessed through `get_hyperparameters`?

Reply:
I agree with removing the instance attributes `n_jobs`, `max_ts_length` and `use_fallback_model`, but I would prefer to keep `_local_model_args` as-is for now.